test/tools/callback-wrapper.js

```diff
@@ -0,0 +1,59 @@
+const createCallbackWrapper = require('../../src/tools/create-callback-wrapper');
+const callbackStatusCatcher = require('../../src/app-middlewares/after/callback-status-catcher');
+let should = require('should');
+
+const CALLBACK_URL = 'http://example.com/callback';
+let input = {
+  _zapier: {
+    event: {
+      callbackUrl: CALLBACK_URL
+    }
+  }
+};
+describe('callbackwrapper', () => {
+  let wrapper;
+
+  before(() => {
```

Comment:
Similar comment in this file - if we can do an `async` function in the `describe` above, let's make a function that calls `createCallbackWrapper` and call that a bunch. way more clear IMO